package-lock.json

```diff
@@ -12167,41 +12167,6 @@
         "node": "^14.15.0 || ^16.10.0 || >=18.0.0"
       }
     },
```

Comment:
issue(blocking): why package-lock.json change without the package.json change. Seems strange ? Did you rebase the branch before making your Pull request

Reply:
Noticed it too, and I did rebase. Can't tell why it triggers a change